source/AI.cpp

```diff
@@ -1773,16 +1775,17 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
-Point AI::StoppingPoint(const Ship &ship, bool &shouldReverse)
+// Instead of coming to a full stop, adjust to a target velocity vector
+Point AI::AdjustPoint(const Ship &ship, const Point &targetVelocity, bool &shouldReverse)
```

Comment:
I don't think "AdjustPoint()" is a very clear name for this function. Maybe just keep the old name of "StoppingPoint()"? Or maybe you can think of a better name.